src/App.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div id="app">
+    <div id="app" align="center">
```

Comment:
should not use inline-styles.
should not add this align-center style to #app, since it's not a global style.